projects/storefrontlib/src/shared/components/spinner/spinner-button.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+@Component({
+  selector: 'cx-spinner-button',
+  templateUrl: './spinner-button.component.html',
+})
+export class SpinnerButtonComponent {
+  @Input()
+  disabled = false;
```

Comment:
I would call this prop `loading`. Button could be also disabled and yet don't show a loader.